### ISSUE TITLE
fc6 layer & unpooling layers?

### ISSUE THREAD

**Issue by SarahQiong** [Joker316701882/Deep-Image-Matting#2]
It looks to me that you are missing the fc6 layer in the encoder and the unpooling layer in the decoder. Please correct me if my understanding is wrong.

p.s. I am interested in implementing deep image matting in tensorflow as well, please send me an email if you are interested in discussing the details of the implementation. Thanks.

**Comment by Joker316701882** [comment]
Yes you are right! Seems I missed the fc6, e.g.14th layer. I will correct it. And I replaced unpooling layer by using deconvolution(There is no unpooling ops in tensorflow). It's OK if we have larger training set. The loss architecture is not correct in this code(but this is based on your understanding). Currently, because of some reason, I have to re-prepare all training data. So the update will be postponed a little. Thank you for your advice!
